src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/index.jsx

```diff
@@ -69,8 +69,9 @@ const VideoSourceWidget = ({
       >
         <FormattedMessage {...messages.videoIdChangeAlert} />
       </ErrorAlert>
-      <Form.Group>
-        <div className="border-primary-100 border-bottom pb-4">
+
+      <div className="border-primary-100 border-bottom pb-4">
+        <Form.Group>
```

Comment:
In this file I fixed a major a11y bug. As you can see from testing playground, the two controls were getting mixed together so screen readers would see "Video ID Video URL" as the name of both inputs, instead of seeing one input called "Video ID" and another called "Video URL"

![a11y error](https://github.com/user-attachments/assets/a650779c-76c1-4275-ab99-ddcb150e2964)

I've really come to appreciate how @testing-library helps catch these a11y issues just by writing regular tests!